pkgs/development/libraries/pipewire/default.nix

```diff
@@ -65,6 +71,9 @@ let
     webrtc-audio-processing_1
     webrtc-audio-processing
   ];
+
+  bluezSupport = stdenv.hostPlatform.isLinux;
```

Comment:
And maybe here? Unless those are available on BSD and you don't want them for whatever reason?